src/property-filter/i18n-utils.ts

```diff
@@ -177,39 +157,45 @@ export function usePropertyFilterI18n(def: I18nStrings & I18nStringsExt): I18nSt
       const formatter = i18n(
         'i18nStrings.removeTokenButtonAriaLabel',
         def.removeTokenButtonAriaLabel,
-        format => () => format(toI18n(token, { ariaLabel: true }))
+        format => () => format({ token__formattedText: formatToken(toFormatted(token)) })
       );
       return formatter?.(toFormatted(token)) ?? '';
     },
     tokenEditorTokenActionsAriaLabel: token => {
       const formatter = i18n(
         'i18nStrings.tokenEditorTokenActionsAriaLabel',
         def.tokenEditorTokenActionsAriaLabel,
-        format => () => format(toI18n(token, { ariaLabel: true }))
+        format => () => format({ token__formattedText: formatToken(toFormatted(token)) })
       );
       return formatter?.(toFormatted(token)) ?? '';
     },
     tokenEditorTokenRemoveAriaLabel: token => {
       const formatter = i18n(
         'i18nStrings.tokenEditorTokenRemoveAriaLabel',
         def.tokenEditorTokenRemoveAriaLabel,
-        format => () => format(toI18n(token, { ariaLabel: true }))
+        format => () => format({ token__formattedText: formatToken(toFormatted(token)) })
       );
       return formatter?.(toFormatted(token)) ?? '';
     },
     tokenEditorAddExistingTokenAriaLabel: token => {
       const formatter = i18n(
         'i18nStrings.tokenEditorAddExistingTokenAriaLabel',
         def.tokenEditorAddExistingTokenAriaLabel,
-        format => () => format(toI18n(token, { ariaLabel: true }))
+        format => () => format({ token__formattedText: formatToken(toFormatted(token)) })
       );
       return formatter?.(toFormatted(token)) ?? '';
     },
     tokenEditorAddExistingTokenLabel: token => {
+      const formattedToken = toFormatted(token);
       const formatter = i18n(
         'i18nStrings.tokenEditorAddExistingTokenLabel',
         def.tokenEditorAddExistingTokenLabel,
-        format => () => format(toI18n(token, { ariaLabel: false }))
+        format => () =>
+          format({
```

Comment:
The `tokenEditorAddExistingTokenLabel` label still requires separate token property label, operator, and value. That is because the operator for this label is not translated to its description text, which is only needed for ARIA labels.